src/nvapi_sys.cpp

```diff
@@ -35,4 +35,46 @@ extern "C" {
 
         return Ok(n);
     }
+
+    NvAPI_Status __cdecl NvAPI_SYS_GetDisplayDriverInfo(NV_DISPLAY_DRIVER_INFO* pDriverInfo) {
+        constexpr auto n = __func__;
+
+        if (nvapiAdapterRegistry == nullptr)
+            return ApiNotInitialized(n);
+
+        if (pDriverInfo == nullptr)
+            return InvalidArgument(n);
+
+        if (pDriverInfo->version != NV_DISPLAY_DRIVER_INFO_VER1 && pDriverInfo->version != NV_DISPLAY_DRIVER_INFO_VER2)
+            return IncompatibleStructVersion(n);
+
+        switch (pDriverInfo->version) {
+            case NV_DISPLAY_DRIVER_INFO_VER1: {
+                auto pDriverInfoV1 = reinterpret_cast<NV_DISPLAY_DRIVER_INFO_V1*>(pDriverInfo);
+                pDriverInfoV1->driverVersion = nvapiAdapterRegistry->GetFirstAdapter()->GetDriverVersion();
+                str::tonvss(pDriverInfoV1->szBuildBranch, str::format(NVAPI_VERSION, "_", DXVK_NVAPI_VERSION));
+                pDriverInfoV1->bIsDCHDriver = 1;              // Assume DHC driver for Windows
+                pDriverInfoV1->bIsNVIDIAStudioPackage = 0;    // Lets not support "Studio Package"
+                pDriverInfoV1->bIsNVIDIAGameReadyPackage = 1; // GameReady Package should be "safe" even if other packages is used
+                pDriverInfoV1->bIsNVIDIARTXProductionBranchPackage = 0;
+                pDriverInfoV1->bIsNVIDIARTXNewFeatureBranchPackage = 0;
+                break;
+            }
+            case NV_DISPLAY_DRIVER_INFO_VER2: {
+                pDriverInfo->driverVersion = nvapiAdapterRegistry->GetFirstAdapter()->GetDriverVersion();
+                str::tonvss(pDriverInfo->szBuildBranch, str::format(NVAPI_VERSION, "_", DXVK_NVAPI_VERSION));
+                pDriverInfo->bIsDCHDriver = 1;              // Assume DHC driver for Windows
+                pDriverInfo->bIsNVIDIAStudioPackage = 0;    // Lets not support "Studio Package"
+                pDriverInfo->bIsNVIDIAGameReadyPackage = 1; // GameReady Package should be "safe" even if other packages is used
+                pDriverInfo->bIsNVIDIARTXProductionBranchPackage = 0;
+                pDriverInfo->bIsNVIDIARTXNewFeatureBranchPackage = 0;
+                str::tonvss(pDriverInfo->szBuildBaseBranch, "DXVK-NVAPI");
```

Comment:
Would something like:
 `str::tonvss(pDriverInfo->szBuildBaseBranch, str::format(NVAPI_VERSION, " (NVAPI Open Source Interface)"));`
make it even more "tidy"? 😄 

It will show as:
`Driver Branch : r535 (NVAPI Open Source Interface)`

on my test proggy...

Reply:
This is a fixed string on NVIDIA's NVAPI, something like "NVIDIA NVAPI Interface 1.0". Since we already have the version number in the branch name, I figured it is fine to use a static thing here too.

Reply:
Let's go for `r5xx` since that seems to also match NVIDIA's NVAPI.